support/dynamics/gp/difference-between-full-limited-and-self-service-user-types.md

```diff
@@ -1,38 +1,45 @@
-description: This article provides the answer for the question about what is the difference between Full, Limited, and Self Service User Types in User Setup in Microsoft Dynamics GP.
-ms.reviewer: cwaswick
-ms.date: 03/31/2021
+title: Difference between Full, Limited, and Self Service user types
+description: This article provides the answer for the question about what is the difference between Full, Limited, and Self Service user types in User Setup in Microsoft Dynamics GP.
+ms.reviewer: theley
+ms.date: 02/20/2024
 ---
-# Difference between Full, Limited, and Self Service User Types in User Setup in Microsoft Dynamics GP
+# Difference between Full, Limited, and Self Service user types in User Setup in Microsoft Dynamics GP
 
-This article introduces the difference between Full, Limited, and Self Service User Types in User Setup in Microsoft Dynamics GP.
+This article introduces the difference between Full, Limited, and Self Service user types in **User Setup** in Microsoft Dynamics GP.
 
 _Applies to:_ &nbsp; Microsoft Dynamics GP  
 _Original KB number:_ &nbsp; 4015051
 
-## Question
+> [!TIP]
+> To open the **User Setup** window, select **Tools** under **Microsoft Dynamics GP**, point to **Setup** > **System**, and select **User**.
 
-In the **User Setup** window, what is the difference between Full, Limited, and Self Service User Types?
+## Introduction
 
-To open this window, select **Tools** under **Microsoft Dynamics GP**, point to **Setup**, point to **System**, and select **User**.
+At a high level, the permissions for each user type are as follows:
 
-## Answer
+- Self Service
 
-At a high level, the permissions for each user Type are as follows:
+  - Ability to create requisitions.
+  - Ability to enter Payroll timecards and expenses.
+  - Ability to enter Project timesheets and expenses, benefits self service, and workflows. (Also known as **Enter** access.)
 
-- Self Service - Provides the following abilities:
-  - Ability to create requisitions
-  - Ability to enter Payroll Timecards and expenses
-  - Ability to enter Project timesheets and expenses, benefits self service,  and workflow. (Also known as Enter access)
+- Limited
 
-- Limited- Can do all the Self Service user can do, plus view all inquiry, reports, and Smartlists. (Also known as Enter and Inquire access)
+  The Limited user can do all operations the Self Service user can do. In addition, the Limited user can view all inquiries, reports, and Smartlists. (Also known as **Enter** and **Inquire** access.)
+- Full
 
-- Full- Has full control in Microsoft Dynamics GP. (Also known as Enter, Inquire, and Edit access)
+  The Full user has full control of Microsoft Dynamics GP. (Also known as **Enter**, **Inquire**, and **Edit** access.)
+
+This is always a grey area because you might expect something it doesn't allow based on the user type. It's recommended to set up a user and test your process if you have limited users.
```

Comment:
What a user type does in Dynamics GP is always a grey area because you might expect something to function in a specific way and the user type does not allow based on the user type specifications. It's recommended to set up a user and test your process if you have limited users.

Reply:
Thank you @theley502, I updated the Line 34 as you suggested!